### ISSUE TITLE
Paget disease of bone  HP:0034159

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11617]
**Preferred term label:** Paget bone lesion
There are currently no synonyms,

**Synonyms**
Paget disease of bone
Paget's disease of bone
Osteitis deformans
PDB

**Definition (free text, please give PubMed ID)**
Paget's disease of bone (PDB) is a chronic progressive metabolic bone disease characterized by abnormal activation of osteoclasts that results in abnormal bone resorption and compensatory osteogenic sclerosis. DB can affect one (monostotic form) or more (polyostotic form) regions of the skeleton, with multiple constitutive symptoms, including bone pain, bone deformity, pathological fracture, tinnitus, hearing loss, vision loss, and osteoarthritis.
PMID:33599011

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
updated
